api/openapi-spec/v1.0.yaml

```diff
@@ -91,6 +91,56 @@ paths:
         default:
           $ref: '#/components/responses/error'
       x-ms-docs-operation-type: operation
+  /v1.0/me/photo:
```

Comment:
The MS Graph spec defines this somewhat different (see https://learn.microsoft.com/en-us/graph/api/profilephoto-get?view=graph-rest-1.0&tabs=http#examples)

To get a photo use:
`GET /v1.0/me/photo/$value`

`PUT` and `PATCH` are also supposed to use ` /v1.0/me/photo/$value`

There are also endpoint for requesting a specific the photo in a specific size, but IIRC  we agreed to ignore those for the first iteration.

`GET /v1.0/me/photo` is only supposed to return metadata of the photo. Not sure if we really need that atm.